README.md

```diff
@@ -126,14 +364,135 @@ The name of an ApplicationPool for this IIS Web Site
+  applicationpool      => 'DefaultAppPool',
+  enabledprotocols     => 'https',
+  bindings             => [
+    {
+      'bindinginformation'   => '*:80:',
+      'protocol'             => 'http',
+    },
+  ],
+}
+```
+
+Use the default HTTPS port
+```puppet
+iis_site { 'mysite':
+  ensure               => 'started',
+  applicationpool      => 'DefaultAppPool',
+  enabledprotocols     => 'https',
+  bindings             => [
+    {
+      'bindinginformation'   => '*:443:',
+      'protocol'             => 'https',
+      'certificatehash'      => '3598FAE5ADDB8BA32A061C5579829B359409856F',
+      'certificatestorename' => 'MY',
+      'protocol'             => 'https',
+      'sslflags'             => 1,
+    },
+  ],
+}
+```
+
+Multiple bindings, one for HTTP and another for HTTPS on non-default port
+```puppet
+iis_site { 'mysite':
+  ensure               => 'started',
+  applicationpool      => 'DefaultAppPool',
+  enabledprotocols     => 'https',
+  bindings             => [
+    {
+      'bindinginformation'   => '*:80:insecure.website.com',
+      'protocol'             => 'http',
+    },
+    {
+      'bindinginformation'   => '*:8443:',
+      'protocol'             => 'https',
+      'certificatehash'      => '3598FAE5ADDB8BA32A061C5579829B359409856F',
+      'certificatestorename' => 'MY',
+      'protocol'             => 'https',
+      'sslflags'             => 1,
+    },
+  ],
+}
+```
+
+Each binding is a hash with the following keys
+
+###### `bindinginformation`
+
+The `bindinginformation` value should be in the form of the IPv4/IPv6 address or wildcard *, then the port, then the optional hostname separated by colons:  `(ip|\*):[1-65535]:(hostname)`
+
+###### `certificatehash`
+
+**Only valid with a protocol of https**
+
+Specifies the thumbprint, also known as the certificatehash, of the certificate that will be used by the IIS site. You can retrieve the thumbprint for a certificate using PowerShell, for example:
+
+```powershell
+PS> Get-ChildItem -Path Cert:\LocalMachine\My
+
+   PSParentPath: Microsoft.PowerShell.Security\Certificate::LocalMachine\My
+
+Thumbprint                                Subject
+----------                                -------
+D4765AA1CE1F25EC29677F87C95D818CE734C2E5  CN=www.webserver.local
+```
+
+###### `certificatestorename`
+
+**Only valid with a protocol of https**
+
+Specifies the certificate store to search for the relevant `certificatehash`.  Typically this `MY` which is the Personal certificate store for the Local Machine.  You can retrieve the list of stores using PowerShell:
+
+```powershell
+PS> Get-ChildItem -Path Cert:\LocalMachine
+
+Name : TrustedPublisher
+
+Name : ClientAuthIssuer
+
+Name : My
+
+...
+```
+
+###### `protocol`
+A value of `http` indicates a binding that uses the HTTP protocol. A value of `https` indicates a binding that uses HTTP over SSL
+
+###### `sslflags`
+
+**Only valid with a protocol of https**
+
+The sslflags parameter accepts integer values from 0 to 3.
```

Comment:
`sslflags` parameter should be in backticks, but the integer values in the following bullet list don't need backticks (or single quotes); just leave them unmarked.